README.md

```diff
@@ -53,7 +52,21 @@ class ExpensiveRenderer extends React.Component {
 }
 ```
 
-## Benchmarking
+## Do I Need `shouldComponentUpdate`?
+
+> What's faster than a really fast deep comparion? No deep comparison at all.
+
+—This Readme
+
+
+Deep checks in React's `shouldComponentUpdate` should not be used blindly.
```

Comment:
Should re-apply the format width to this paragraph (and I _do_ like to have lines broken at 80 or 100 or whatever in MD text). Lots of plugins to do this if you don't want to manually...